### PR TITLE
Getting started!

### PR DESCRIPTION
**Pull request by jacquesbh**
This is a suggestion.

Of course we can't accept all plugins. And we need to keep a high quality
codebase.  
Also, having some rules is important for that I think.

I think it could be possible to add some, like:

- SHOULD have a scrutinizer notation in the main README.
- SHOULD be maintained and stay up-to-date with the latest version of Sylius.

And so on…

I've specified that all plugins must follow the Software Craftmanship approach.  
It's pretty simple: the conception of the plugin has to be high quality too,
along with the source code.

In some other words, we can't accept plugins which are just done quickly to
fullfill a need. That's not how craftmen (and women) do their job.

### CODE REVIEW

File: plugins/_template.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# {{Plugin's name}}
+
```
Comment (oallain) [inline plugins/_template.md:2]: IMO, we need to add Pluging repository link here.

File: plugins/_template.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+{{Plugin's description}}
```
Comment (oallain) [inline plugins/_template.md:3]: In description you include list of features ?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+If you have a plugin that you want to appear on this list, please open a PR.  
+We will review the plugin and give an opinion about it. If the opinion is
+favorable then we will update your PR with our comment and merge it.
+
+Your plugin MUST be high quality. We accept only plugins with a code quality
+of 9.80 out of 10 and above.
+
+Also it has to be:
```
Comment (oallain) [inline CONTRIBUTING.md:10]: We add a word about Tests ?
No need for 100% test coverage, but plugin have the minimum testing that covers major functionality?

**Review by oallain — COMMENTED** [review]
I think these are a good foundation.
Here are my few remarks.

File: plugins/_template.md (hunk 2 above)
Comment (jacquesbh) [reply plugins/_template.md:3]: You can put whatever you want ^^.

**Review by oallain — APPROVED** [review]

**Review by maximehuran — APPROVED** [review]

**Comment by oallain** [comment]
Thanks @jacquesbh :rocket:
